app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -11,10 +12,9 @@ class AuthorizationController < ApplicationController
 
     def index
       return confirm_two_factor_authenticated(request_id) unless user_fully_authenticated?
-      return redirect_to verify_url if identity_needs_verification?
+      return if idv_needed_or_in_process
```

Comment:
I would expect this method to return true or false and for it to end in a question mark, but this method is redirecting, which is confusing. When I see `return if [some condition is true]`, I expect the method to be exited without any side effects.

Maybe something like this?

```ruby
return redirect_to_profile_or_verify_profile_url if profile_or_identity_needs_verification?

def redirect_to_profile_or_verify_profile_url
  return redirect_to profile_or_verify_profile_url if profile_needs_verification?
  redirect_to verify_url if identity_needs_verification?
end

def profile_or_identity_needs_verification?
  profile_needs_verification? || identity_needs_verification?
end

```